addons/purchase/models/purchase.py

```diff
@@ -657,7 +657,7 @@ class PurchaseOrderLine(models.Model):
 
     partner_id = fields.Many2one('res.partner', related='order_id.partner_id', string='Partner', readonly=True, store=True)
     currency_id = fields.Many2one(related='order_id.currency_id', store=True, string='Currency', readonly=True)
-    date_order = fields.Datetime(related='order_id.date_order', string='Order Date', readonly=True)
+    date_order = fields.Datetime(related='order_id.date_order', string='Order Date', readonly=True, store=True)
```

Comment:
where does that diff come from?

Reply:
I had a db error during my dev when it didn't find the field in DB. Did not realise that after code modifications it was not need anymore.